procfile/procfile.go

```diff
@@ -47,11 +95,16 @@ func Parse(r io.Reader) (Procfile, error) {
 
 // ParseProcfile takes a byte slice representing a YAML Procfile and parses it
 // into a Procfile.
-func ParseProcfile(b []byte) (Procfile, error) {
-	p, err := parseStandardProcfile(b)
+func ParseProcfile(b []byte) (p Procfile, err error) {
+	p, err = parseStandardProcfile(b)
```

Comment:
Yeah, definitely was thinking about it. The only reason I didn't was because it means we'd probably want to nest the processes. e.g.

```yaml
version: 'v1'
processes:
  web:
    command: ./bin/web
```

which would be fine, but mostly wanted to avoid it if possible. I figure if the time comes where we need a v3, then we can do that and v1/v2/v3 will still be easy to differentiate.